### PR TITLE
Basic proof-of-concept app

### PR DESCRIPTION
**Pull request by hangler**
This PR represents the NJSA app at a very basic level of functionality. The PSA CSV extracts can be POSTed and new employees inserted to the NJSA + CallWeb databases. The employees can be reviewed and their statuses can be updated (with CallWeb updates as appropriate), and the various survey dates can be adjusted in the admin interface. A manual reconciliation pulling from CallWeb can be run.

Major functionality still to come before this PR should be merged:

* More code documentation.
* Tests.
* Better error handling on edge cases.
* Build / deploy configs for smoother OpenShift deployment.
* Acceptance of JSON, not just CSV, for creating employees.
* Scripts (ready to run as OpenShift jobs) for loading the PSA API and reconciling employees on a regular basis.
* Various UI updates.

### CODE REVIEW

File: ClientApp/.env [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# No trailing slash
+REACT_APP_APP_DOMAIN='http://localhost:3000'
+REACT_APP_API_DOMAIN='http://localhost:5050'
+
+# MUST have a trailing slash!
+REACT_APP_APP_PATH='/'
+
+REACT_APP_AUTH_RESPONSE_TYPE='code'
+REACT_APP_AUTH_SCOPE='openid profile resourceApi'
+REACT_APP_AUTH_URL='https://dev.oidc.gov.bc.ca/auth/realms/ytaqhqia'
```
Comment (bdolor) [inline ClientApp/.env:14]: at the very least, the values here wouldn't be hard coded, in some cases because they might be sensitive and/or are better served as variables that can be set depending on which environment it's being deployed to.

Comment (bdolor) [reply ClientApp/.env:14]: might even be better to consider adding `.env` to the `.gitignore` and add an example template and instructions in the readme.

**Review by bdolor — CHANGES_REQUESTED** [review]

File: ClientApp/.env (hunk 1 above)
Comment (hangler) [reply ClientApp/.env:14]: Tracking in #35.

Comment (hangler) [reply ClientApp/.env:14]: PR: https://github.com/bcgov/NewJobSurveyAdmin/pull/36

**Review by bdolor — APPROVED** [review]
